build.ab

```diff
@@ -13,16 +6,6 @@ $test -f "Cargo.toml" > /dev/null$ failed {
 
 unsafe {
     if status == 0 {
-        let target_str = ""
-        loop target in targets {
-            target_str += "--target {target} "
-        }
-        $cargo build --release {target_str}$
-        // Move ambers to the release directory
-        $mv target/aarch64-apple-darwin/release/amber target/release/amber_macos_aarch64$
-        $mv target/aarch64-unknown-linux-musl/release/amber target/release/amber_linux_aarch64$
-        $mv target/x86_64-apple-darwin/release/amber target/release/amber_macos_x86_64$
-        $mv target/x86_64-unknown-linux-musl/release/amber target/release/amber_linux_x86_64$
```

Comment:
I used to use this script to recompile binaries. Now they are compiled in the cloud hence the code deletion.